package.json

```diff
@@ -155,7 +157,7 @@
     },
     "husky": {
         "hooks": {
-            "pre-commit": "run-p changelog schema schema-higlass schema-theme schema-template format && git add .",
```

Comment:
Sometimes I only want to commit things that are staged. When the pre-commit hook would run, it would add all my unstaged files

Reply:
Ah you're right. I'll add it back.

Reply:
@manzt Do you aware of any better way to deal with this?

Reply:
I think this is something we could probably run / check in CI? E.g., We could have a GitHub action run an NPM script and assert that there are no git changes. If there are, fail in CI and require the command is run locally. 

I personally really dislike pre-commit hooks because I commit often.

Reply:
I like the idea! I also dislike waiting for the pre-commit hooks to run scripts.